routes/web.php

```diff
@@ -21,8 +22,10 @@
 Route::resources([
     'courses' => CourseController::class,
     'courseusers' => CourseUserController::class,
+    'attendance' => AttendanceController::class,
 ]);
 
+Route::get('/studentattendance', [AttendanceController::class, 'register'])->name('attendance.register');
```

Comment:
Rename this to /student/attendance. The way it is named currently isn't the conventional way to name routes